components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -101,18 +101,28 @@ export class ProjectEffects {
   @Effect()
   createProject$ = this.actions$.pipe(
       ofType(ProjectActionTypes.CREATE),
-      mergeMap(({ payload: { id, name } }: CreateProject) =>
-      this.requests.createProject(id, name).pipe(
-        map((resp: ProjectSuccessPayload) => new CreateProjectSuccess(resp)),
-        catchError((error: HttpErrorResponse) => observableOf(new CreateProjectFailure(error))))));
+      mergeMap(({ payload: { id, name, skip } }: CreateProject) =>
+        this.requests.createProject(id, name, skip).pipe(
+          map((resp: ProjectSuccessPayload) => new CreateProjectSuccess({resp, skip})),
+          catchError((error: HttpErrorResponse) =>
+            observableOf(new CreateProjectFailure(error))))));
 
   @Effect()
   createProjectSuccess$ = this.actions$.pipe(
       ofType(ProjectActionTypes.CREATE_SUCCESS),
-      map(({ payload: { project } }: CreateProjectSuccess) => new CreateNotification({
-      type: Type.info,
-        message: `Created project ${project.id} and associated policies.`
-    })));
+      map(({ payload: { resp, skip } }: CreateProjectSuccess) => {
+        if (skip) {
+          return new CreateNotification({
+            type: Type.info,
+            message: `Created project ${resp.project.id}.`
+          });
+        } else {
+          return new CreateNotification({
+            type: Type.info,
+            message: `Created project ${resp.project.id} and associated policies.`
+          });
+        }
```

Comment:
Could be more concise:
```suggestion
      return new CreateNotification({
        type: Type.info,
        message: skip
          ? `Created project ${resp.project.id}.`
          : `Created project ${resp.project.id} and associated policies.`
      });
 
```